pylint/checkers/typecheck.py

```diff
@@ -72,7 +72,7 @@
 from typing import Any, Callable, Iterator, List, Optional, Pattern, Tuple
 
 import astroid
-from astroid import bases, nodes
+from astroid import bases, exceptions, nodes
```

Comment:
I'm not sure we should use this import here.
There is a `pylint.exceptions` package too which is often imported with
```py
from pylint import exceptions
```
Thus doing the same for astroid will lead to name conflicts.

At some point @Pierre-Sassoulas and I discussed this I think, although we didn't reach a conclusion for a good name. For the time being, I would recommend this
```py
import astroid.exceptions

try:
    ...
except astroid.exceptions...:
    ...
```
It's a bit lengthy but already used elsewhere. Should be good enough util we find something else.

Reply:
I have added an `import astroid.exceptions` line. There is already a `import astroid` statement, but `astroid.exceptions.StatementMissing` could not be resolved by VS Code (/ Pylance).. I think because of the wildcard import in `__init__.py` in `astroid`.